modules/ROOT/pages/branding-and-customization.adoc

```diff
@@ -4,98 +4,101 @@ include::_attributes.adoc[]
+Select *Community Builder* in the ACM administrator interface. By default, 
+the Community Builder displays the *Home* page. You can select the *Home* drop-down list arrow to display a list 
+of community pages and specify a different page for customization.
 
-== Settings
-
-Click the gear icon at the top left to access Settings:
-
-* *General*: These settings include *Community Title*, *Public Access* to make the page public, and *Topics* that define the structure of the site, such as *Navigational Topics* and *Featured Topics*. Topics are a mechanism to organize content such as CMS articles and forum posts.
-* *Theme*: Here you can manage and configure the theme of your site.
-* *Languages*: Manage the languages for multilingual communities and configure translation options.
-* *SEO*: Search Engine Optimization (SEO) features define what information can be indexed by external services and discovered through online searches, and can increase traffic to your public community.
-* *CMS Connect*: Connect to an external Content Management System (CMS) so you can embed assets from the CMS in your community.
-* *Advanced*: Advanced settings include configuring Google Analytics, setting the `<head>` markup on each community page, configuring the components panel, and optimizing performance.
-* *Security*: Manage security of your community.
-* *Developer*: Developer options, including exporting a page, a community theme, or an entire community template.
-* *Updates*: Manage Community Cloud updates.
-
-== Themes
+The Community Builder interface contains customization icons as described in the following sections. 
 
-Click the paintbrush icon at the top left to access Theme options:
+== Components
 
-* *Colors*: The site's *General* color scheme includes *Text Color*, *Detail Text Color*, *Action Color*, *Link Color*, *Overlay Text Color*, *Border Color*, and *Page Background Color*. *Login Pages* have their own color scheme which includes *Background Color*, *Card Background Color*, and *Error Text Color*.
-* *Images*: Set the site's *Company Logo* and the login page's *Background Image*.
-* *Fonts*: The site's fonts include the *Primary Font* and *Header Fonts*. You can also set the site's *Text Case*.
-* *Theme Settings*: The site's theme settings include *Show header*, *Fix the header's position*, *Show hero* to enable the site's hero image, *Start hero under header*, and *Set max page width* in pixels.
+Select the *Components* lightning bolt icon in the top left navigation to display the component palette. 
+The component palette includes Salesforce Community Cloud and ACM-specific components. 
+Drag and drop the desired components from the component palette to the community page. Each 
+component has its own configuration options that you can then access by selecting the component on the page.
 
-== Theme Navigation Menu
+For additional component information, refer to the Salesforce Community Cloud and ACM product documentation.
```

Comment:
please add links here to: 
community cloud components docs: https://help.salesforce.com/articleView?id=rss_component_koa_kokua_components.htm&type=5
ACM components: link to the corresponding page in our docs

Reply:
I've updated this  - thank you! @valkyrie69 - can you please check the relative link syntax?

Reply:
The relative link syntax is correct, but the relative link should be to the components page and not the index page. Fixed.

That Salesforce link is for Koa and Kokua components, and that page has a note saying those were replaced in Summer '17 by the Lightning Components, and I know ACM uses Lightning Components, so I changed the link to the Lightning Components page.